Makefile

```diff
@@ -294,6 +310,7 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	$(MAKE) bundle-reset-date
```

Comment:
Why reset the date here ?

Reply:
Updating [operator-sdk to v1.26.0](https://github.com/medik8s/node-maintenance-operator/pull/!65) resulted in setting the **createdAt** field after running [`make bundle`](https://github.com/operator-framework/operator-sdk/commit/fe1e18c1b96c2b47d51c1cd33c0dc564eb66ebfd#diff-5f7bc62652b60bcf694b6fb39b2114b24bfa6559ac7f31969c12ee6034cec3a9).
[In the past](https://github.com/medik8s/node-healthcheck-operator/pull/89#discussion_r803535257) we didn't see a good use for setting this field, since it will create changes every time we run `make bundle`.
Therefore, I suggest of resetting the createdAt field.